Stdlib doesn't build when enabling SWIFT_TASK_DEBUG_LOG_ENABLED

### Description

_No response_

### Reproduction

Enable [this line](https://github.com/swiftlang/swift/blob/9294297356cd7fdf3c7787503059ec6013acc879/stdlib/public/Concurrency/TaskPrivate.h#L54), then try to build. You get this:

```
swift/stdlib/public/Concurrency/TaskGroup.cpp:1819:56: error: no member named 'to_string' in 'std::atomic<unsigned long long>'
 1819 |     SWIFT_TASK_DEBUG_LOG("poll, after CAS: %s", status.to_string().c_str());
      |                                                 ~~~~~~ ^
swift/stdlib/public/Concurrency/TaskGroup.cpp:2164:31: warning: format specifies type 'int' but the argument has type 'uint64_t' (aka 'unsigned long long') [-Wformat]
 2162 |   SWIFT_TASK_DEBUG_LOG("add pending %s to group(%p), tasks pending = %d",
      |                                                                      ~~
      |                                                                      %llu
 2163 |                        unconditionally ? "unconditionally" : "",
 2164 |                        group, assumed.pendingTasks(group));
      |                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~
swift/stdlib/public/Concurrency/TaskPrivate.h:59:35: note: expanded from macro 'SWIFT_TASK_DEBUG_LOG'
   57 |   fprintf(stderr, "[%#lx] [%s:%d](%s) " fmt "\n",                              \
      |                                         ~~~
   58 |           (unsigned long)Thread::current().platformThreadId(), __FILE__,       \
   59 |           __LINE__, __FUNCTION__, __VA_ARGS__)
      |                                   ^~~~~~~~~~~
```

### Expected behavior

Builds successfully

### Environment

Swift version 6.5-dev (LLVM f38384c5a9a8900, Swift 5b92e5c57d00586)
Target: arm64-apple-macosx15.0

### Additional information

_No response_

--- a/stdlib/public/Concurrency/TaskGroup.cpp
+++ b/stdlib/public/Concurrency/TaskGroup.cpp
@@ -1816,7 +1816,7 @@ reevaluate_if_taskgroup_has_results:;
       assumed = TaskGroupStatus{assumedStatus};
       continue; // We raced with something, try again.
     }
-    SWIFT_TASK_DEBUG_LOG("poll, after CAS: %s", status.to_string().c_str());
+    SWIFT_TASK_DEBUG_LOG("poll, after CAS: %s", statusString().c_str());
 
     // We're going back to running the task, so if we suspended before,
     // we need to flag it as running again.
@@ -2159,9 +2159,9 @@ SWIFT_CC(swift)
 static bool swift_taskGroup_addPendingImpl(TaskGroup *_group, bool unconditionally) {
   auto group = asBaseImpl(_group);
   auto assumed = group->statusAddPendingTaskAssumeRelaxed(unconditionally);
-  SWIFT_TASK_DEBUG_LOG("add pending %s to group(%p), tasks pending = %d",
-                       unconditionally ? "unconditionally" : "",
-                       group, assumed.pendingTasks(group));
+  SWIFT_TASK_DEBUG_LOG("add pending %s to group(%p), tasks pending = %llu",
+                       unconditionally ? "unconditionally" : "", group,
+                       assumed.pendingTasks(group));
   return !assumed.isCancelled();
 }
 

